pkg/api/errors/scheduler.go

```diff
@@ -22,86 +22,63 @@ import (
 	kiterrors "github.com/dapr/kit/errors"
 )
 
-const (
-	CodePrefixScheduler               = "SCHEDULER_" // TODO(Cassie): move this to kit eventually
-	InFixJob            ReasonSegment = "JOB_"
-	InFixAppID          ReasonSegment = "APPID_"
-	InFixGet            ReasonSegment = "GET_"
-	InFixList           ReasonSegment = "LIST_"
-	InFixDelete         ReasonSegment = "DELETE_"
-	InFixSchedule       ReasonSegment = "SCHEDULE_"
-	PostFixRepeats      ReasonSegment = "REPEATS"
-	PostFixJob          ReasonSegment = "JOB"
-	PostFixJobs         ReasonSegment = "JOBS"
-
-	MsgScheduleJob = "failed to schedule job"
-	MsgGetJob      = "failed to get job"
-	MsgListJobs    = "failed to list jobs"
-	MsgDeleteJob   = "failed to delete job"
-)
```

Comment:
Moved to `pkg/messages/errorcodes/errorcodes.go` to align with other error codes